packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -177,7 +175,10 @@ export default class DCWebxdc {
         const ses = sessionFromAccountId(accountId)
         accounts_sessions.push(accountId)
         ses.protocol.handle('webxdc', (...args) =>
-          webxdcProtocolHandler(this.rpc, ...args)
+          webxdcProtocolHandler(this.rpc, appId, ...args)
+        )
+        ses.protocol.handle('maps', (...args) =>
```

Comment:
So, can't we ensure that only the maps .xdc can access this protocol, by making a separate `ses`?
Or there should be another way.

Otherwise this is a pretty concerning attack vector.

Reply:
Ok I see, there is no app specific session but only an account specific. Yes then it makes sense to have a separate session for the maps.xcd

Reply:
I choosed a different approach: the maps:// protocol is now restricted by CSP: only apps with internet access are now allowed to add image sources with maps://* as target.

I tried the separate sessions approach but somehow the partition persist:webxdc_<accountId>.<appId> resulted in errors. Maybe there is some magic relying on the format persist:webxdc_<accountId> ??
Tried it here: #5455